pyproject.toml

```diff
@@ -39,6 +39,7 @@ sentry-sdk = "^2.16.0"
 html5lib = "^1.1"
 beautifulsoup4 = "^4.12.3"
 django-xff = "^1.4.0"
+markdown = "^3.6"
```

Comment:
This is already added as a dependency of wagtail-markdown.

Reply:
> What is the benefit of using markdown for the title

I went with a `CharField` mostly to prevent the ability of adding line breaks. It would be much more confusing to be able to add line breaks in the page but not have it reflect on the page.

If I were to go with the rich text field (which admittedly was my first idea & implementation), the output would be: `<p data-block-key="hs0yc">Some content <b>here</b></p>`

With the use of markdown, the output is: `<p>Some content <strong>here</strong></p>`.

It was easier to get rid of the paragraph tags from the markdown because it's just removing `<p>` and `</p>`. While with the rich text output, we actually need to either parse the output, find the paragraph tag and remove it, or use regex to remove the tags.

... A simpler solution might actually be to just have 2 hero heading fields (`hero_heading_1` and `hero_heading_2`). In hindsight, this is probably the least confusing option. I'm gonna go ahead and do that.

> ... and rich text for the pink / purple text?

I wanted to give the editor the option of having multiple paragraphs for the description.

Reply:
> ... A simpler solution might actually be to just have 2 hero heading fields (`hero_heading_1` and `hero_heading_2`). In hindsight, this is probably the least confusing option. I'm gonna go ahead and do that.

I've gone ahead and done that.

<details><summary>Screenshot</summary>

![image](https://github.com/user-attachments/assets/dc1d0a16-d7f9-41f3-84ef-59121afdf7dd)

</details>

Reply:
This sounds a good approach